addons/website_livechat/models/discuss_channel.py

```diff
@@ -51,7 +45,9 @@ def _to_store_defaults(self, for_current_user=True):
                 predicate=lambda channel: channel.livechat_visitor_id
                 and self.livechat_visitor_id.has_access("read"),
             ),
-            "requested_by_operator",
+            Store.Attr(
+                "requested_by_operator", predicate=lambda channel: channel.livechat_visitor_id
```

Comment:
for similar reasons than other PRs, predicate should be on livechat type instead